### PR TITLE
Bug 1569720 Add steps to verify joined networks

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1569720

### CODE REVIEW

**Comment by bmeng** [comment]
Doc change looks good. QE reviewed.

**Comment by vikram-redhat** [comment]
@bmcelvee - this was verified 15/Jun and I think should be merged. Nothing else holding it up?

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL

File: admin_guide/managing_networking.adoc [hunk 1]
```diff
@@ -52,6 +52,16 @@ service named `db` in a project `myproject`, use `db.myproject.svc.cluster.local
 Alternatively, instead of specifying specific project names, you can use the
 `--selector=<project_selector>` option.
 
+To verify which networks you have joined, run:
```
Comment (bfallonf) [inline admin_guide/managing_networking.adoc]: Can I suggest s/To verify which networks you have joined, run:/To verify the networks you have joined together:

I read how it is now as "I have joined on these networks" like "I have joined the meeting" and that's not the same.

**Comment by bfallonf** [comment]
@bmcelvee I have one nit, but LGTM otherwise :cake:

File: admin_guide/managing_networking.adoc (hunk 1 above)
Comment (bmcelvee) [reply admin_guide/managing_networking.adoc]: Sound good to me! Thanks, @bfallonf! :ice_cream:

**Pull request merged**